File: multi_livechat/models/demo/res_users_settings.py
# Copyright 2022 Ivan Yelizariev <https://twitter.com/yelizariev>
# License MIT (https://opensource.org/licenses/MIT).
from odoo import fields, models


class ResUsersSettings(models.Model):
    _inherit = "res.users.settings"

    is_discuss_sidebar_category_echo_demo_open = fields.Boolean(
        "Is category ECHO (Demo) open", default=True
    )

File: multi_livechat/static/src/demo/models/res_users_settings.js
/** @odoo-module **/

import { registerPatch } from "@mail/model/model_core";
import { attr } from '@mail/model/model_field';

/**
 * Mirrors the fields of the python model res.users.settings.
 */
registerPatch({
    name: 'res.users.settings',
    fields: {
        is_discuss_sidebar_category_line_open: attr({
            default: true,
        }),
    },
});
